### PR TITLE
save session and solution

### LINKED ISSUE #10: Implement saving models

### PR DESCRIPTION
**Pull request by dharanisakary**
Changes for saving the model and logging start session. Please let me know for any issues.

Resolves #10 

Accurate diff here: https://github.com/Dragoon-Lab/topomath/compare/master...save_model

### CODE REVIEW

**Comment by jonwetzel** [comment]
Accurate diff here: https://github.com/Dragoon-Lab/topomath/compare/master...save_model

File: www/login.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+					<input id="problem" type="text" name="p" value="rabbits">
+				</div>
+
+				<div class="fieldgroup">
+					<label for="section">Section: </label>
+					<input id="section" type="text" name="s" value="login.html">
+				</div>
+
+				<div class="fieldgroup">
+					<label for="group">Folder:<br> (i.e. group; leave blank for published model) </label>
```
Comment (jonwetzel) [inline www/login.html]: The fields which used to be called "group" should be called "folder" in topomath.  Please go ahead and change the id of this field to "folder" for consistency.

**Comment by jonwetzel** [comment]
@dharanisakary  @sachingrover211 

I can open rabbits in author mode multiple times and it saves and loads even though I didn't specify a folder.  I'm not sure if it's correct for this to be possible, but it's useful for now since we don't have create node--rabbits is the only thing we can effectively test.

**Comment by sachingrover211** [comment]
I think this is good to go.

**Comment by dharanisakary** [comment]
@jonwetzel , updated the Id of field as 'folder' instead of 'group'

File: www/login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	<script type="text/javascript">
+
+	function validateForm(){
+		var form = document.forms["problem-select"];
+		var x = form.u.value;
+		if (x==null || x==""){
+			alert("Please enter a username.");
+			return false;
+		}      
+		// If group field is empty, remove from form.
```
Comment (jonwetzel) [inline www/login.html]: This comment refers to the old group field as well, and uses "g" as the variable in line 22.  I think that  needs to be "f" instead.  Replace these references to "group" with "folder" (including the comments) and replace "g" with "f" (including in the console log.)

File: www/login.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+					<input id="problem" type="text" name="p" value="rabbits">
+				</div>
+
+				<div class="fieldgroup">
+					<label for="section">Section: </label>
+					<input id="section" type="text" name="s" value="login.html">
+				</div>
+
+				<div class="fieldgroup">
+					<label for="folder">Folder:<br> (i.e. group; leave blank for published model) </label>
```
Comment (jonwetzel) [inline www/login.html]: You can remove the reference to group; we won't call folders "groups" in any part of topomath.

**Pull request merged**